pkg/action/env.container.go

```diff
@@ -168,19 +170,10 @@ func getCurrentUser() string {
 	return curuser
 }
 
-func genContainerName(a *Action, prefix string, existing map[string]struct{}) string {
+func genContainerName(a *Action, prefix, suffix string, existing map[string]struct{}) string {
 	// Replace command name "-", ":", and "." to "_".
 	var rpl = strings.NewReplacer("-", "_", ":", "_", ".", "_")
-	base := prefix + rpl.Replace(a.ID)
-	name := base
-	if len(existing) > 0 {
-		_, ok := existing[name]
-		// Set suffix if container already exists.
-		for ; ok; _, ok = existing[name] {
-			name = base + "_" + namesgenerator.GetRandomName(0)
-		}
-	}
-	return name
+	return prefix + rpl.Replace(a.ID) + "_" + suffix
```

Comment:
Considering the above suggestion, always return as `return prefix + rpl.Replace(a.ID) + "_" + namesgenerator.GetRandomName(0)`